packages/vector_graphics_compiler/lib/src/svg/node.dart

```diff
@@ -125,9 +125,20 @@ class ParentNode extends Node {
   }
 
   @override
-  void addPaths(DrawCommandConcatentor addDrawPath, AffineMatrix transform) {
+  void addPaths(DrawCommandBuilder builder, AffineMatrix transform) {
+    final Color? color = paint?.fill?.color;
+
+    if (color != null && color.o < 255) {
```

Comment:
technically I think we'd need to also change so that inheritance only sees the non-opaque color.

Then for now we could write an optimizer that checks if there is only one path under the group and deletes the node and re-applies the opacity

Reply:
That would be a simpler optimizer to write, but I'm not sure how often it would actually help with things. What we really want is an optimizer that gets rid of overdraw when it can and converts semi-transparent overlaps into independent blended triangles.

The thing I'm really not sure about is how we'll deal with seams...

Reply:
it would help more than nothing and take about 5 minutes to write

Reply:
Could also follow that up with a similar optimization that checks if shapes are non-overlapping. That will handle all the easy cases

From there, we have the actual difficult optimization of overdraw elimination.

I worry that for seams we need to write our own shaders to handle